phase1-coordinator/src/coordinator.rs

```diff
@@ -277,7 +278,7 @@ impl From<CoordinatorError> for anyhow::Error {
 /// for mocking system time during testing.
 pub trait TimeSource: Send + Sync {
     /// Provide the current time now in the UTC timezone
-    fn utc_now(&self) -> DateTime<Utc>;
+    fn now_utc(&self) -> OffsetDateTime;
```

Comment:
Is there a reason for this renaming? (Not a blocker or anything, just wondering why it was renamed)

Reply:
I think it's just realigning the internal api with the wrapper function there for consistency.